packages/atomic/src/components/commerce/atomic-commerce-interface/atomic-commerce-interface.spec.ts

```diff
@@ -390,6 +391,48 @@ describe('AtomicCommerceInterface', () => {
     });
   });
 
+  describe('mobile breakpoint', () => {
```

Comment:
The atomic-commerce-interface unit tests do not follow our standards / conventions.

However, refactoring this would be a huge scope creep, so I simply added tests in the same style here.

We have a separate Jira for adjusting the structure of the tests: https://coveord.atlassian.net/browse/KIT-4386.